chainerx_cc/chainerx/python/error.cc

```diff
@@ -20,6 +20,14 @@ void InitChainerxError(pybind11::module& m) {
     py::register_exception<NotImplementedError>(m, "NotImplementedError");
     py::register_exception<GradientError>(m, "GradientError");
     py::register_exception<GradientCheckError>(m, "GradientCheckError");
+
+    py::register_exception_translator([](std::exception_ptr p) {
+        try {
+            if (p) std::rethrow_exception(p);
```

Comment:
Let's use curly braces and linebreaks. I think most of our code follows this style.

Reply:
Fixed.

Reply:
Additionally, fixed to avoid implicit cast from `std::exception_ptr` to `bool`.